httpserver/src/test/kotlin/TestData.kt

```diff
@@ -0,0 +1,68 @@
+import de.fxlae.typeid.TypeId
+import tbdex.sdk.protocol.models.*
+import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.methods.dht.DidDht
+import java.time.OffsetDateTime
+
+object TestData {
```

Comment:
Would it make sense to refine all these messages in separate json files? That way, it's easier to promote them to test-vectors later, which can be used to ensure implementations in other languages work.

Reply:
Yeah, this needs to be done across all the packages so created an issue for it here: https://github.com/orgs/TBD54566975/projects/29/views/1?pane=issue&itemId=47929204